tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml

```diff
@@ -80,6 +71,6 @@ items:
     name: {{ image_set|default("rhods-openshift") }}
     namespace: rhods
   spec:
-    releaseImage: {{ openshift_image|default("quay.io/openshift-release-dev/ocp-release:4.10.18-x86_64") }}
+    releaseImage: {{ openshift_image|default("quay.io/openshift-release-dev/ocp-release:4.10.42-x86_64") }}
```

Comment:
isn't there a way to fetch this value automatically?

Reply:
We are using the variable openshift_image to add the value manually or dynamically. To fetch this value automatically, we can use the API to get the last version in the stable channel and use that value, but since we are testing in different channels with different OpenShift versions, that variable is better managed in the infra file. I'm using a default value but maybe it would be better to delete the default value to avoid confusion. What do you think?